datalake_ingester/queue.py

```diff
@@ -31,7 +30,7 @@ def __init__(self, queue_name, handler=None):
 
     @classmethod
     def from_config(cls):
-        queue_name = get_config_var('queue')
+        queue_name = os.environ.get('DATALAKE_INGESTION_QUEUE')
```

Comment:
Its more like a query I have. Are these environment variables in the DEFAULT_CONFIG file and set using the load_config function?